### PR TITLE
Fixes the Misleading Error Message ( As seen in issue #1811)

### LINKED ISSUE #1811: Misleading error message on malformed formList XML
https://github.com/opendatakit/collect/blob/7d725d760e71e95467f3f2fc847e9638a55dabb9/collect_app/src/main/java/org/odk/collect/android/tasks/DownloadFormListTask.java#L221

1. The code is actually checking tags _and values_ so the error may be triggered with a tag present but no value in the tag.
2. `formId` should be `formID`.

### PR DESCRIPTION
**Pull request by raks097**
Closes #1811 

#### What has been done to verify that this works as intended?
I have run this on my physical Android device and works as intended.
Device Specs:
Sony Xperia C4 Dual E5363 ( Android 6.0)

#### Why is this the best possible solution? Were any other approaches considered?
The previous error message had certain ambiguity in the context of the tag names used in the
message.Those have been changed so that any error message is crisp and clear.

#### Are there any risks to merging this code? If so, what are they?
No.

#### Do we need any specific form for testing your changes? If so, please attach one.
No

### CODE REVIEW

File: collect_app/src/main/java/org/odk/collect/android/tasks/DownloadFormListTask.java [hunk 1]
```diff
@@ -217,8 +217,8 @@ private boolean isXformsListNamespacedElement(Element e) {
                 }
                 if (formId == null || downloadUrl == null || formName == null) {
                     String error =
-                            "Forms list entry " + Integer.toString(i)
-                                    + " is missing one or more tags: formId, name, or downloadUrl";
+                            "Forms List Entry " + Integer.toString(i)
+                                    + " has missing or empty tag(s) : formID ,downloadUrl or name";
```
Comment (lognaturel) [inline collect_app/src/main/java/org/odk/collect/android/tasks/DownloadFormListTask.java]: The project uses American English conventions so there should be no space before the colon. There should be no space before the comma and a space after it.

File: collect_app/src/main/java/org/odk/collect/android/tasks/DownloadFormListTask.java [hunk 2]
```diff
@@ -272,8 +272,8 @@ private boolean isXformsListNamespacedElement(Element e) {
                     }
                     if (downloadUrl == null || formName == null) {
                         String error =
-                                "Forms list entry " + Integer.toString(i)
-                                        + " is missing form name or url attribute";
+                                "Forms List Entry " + Integer.toString(i)
+                                        + " has missing or empty tag(s) : form or url";
```
Comment (lognaturel) [inline collect_app/src/main/java/org/odk/collect/android/tasks/DownloadFormListTask.java]: No space before colon. `url` is an attribute, not a tag. So something like "has missing or empty form name or url attribute" would be better

**Review by lognaturel — CHANGES_REQUESTED** [review]

**Comment by raks097** [comment]
Hey @lognaturel 
I pushed a different branch I was working on to fix this issue.
I'm having troubles reverting back
I'll try my level best to revert back or I'll close this PR and put up a new one.
Sorry for causing this confusion.

**Comment by raks097** [comment]
@lognaturel 
I'm rebasing my branch and will create a new PR.

**Pull request closed without merging**